benchmarks/test_power_nu.py

```diff
@@ -33,7 +34,15 @@ def test_power_nu(model):
     data_lin = np.loadtxt("./benchmarks/data/model%d_pk_nu.txt" % (model+1))
     k_lin = data_lin[:, 0] * cosmo['h']
     pk_lin = data_lin[:, 1] / (cosmo['h']**3)
-    pk_lin_ccl = ccl.linear_matter_power(cosmo, k_lin, a)
+
+    with warnings.catch_warnings():
+        # Linear power with massive neutrinos raises a warning.
+        # Ignore it.
+        # XXX: Do you really want to be raising a warning for this?
+        #      This seems spurious to me.  (MJ)
```

Comment:
I tend to agree with these (and have done so for a while), so I'll open an issue.

Reply:
Yeah, we do probably overwarn on this, we probably need to go through and reconsider how we handle this. Thanks for raising the issue.